app/src/main/res/drawable/bg_spinner_left.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <layer-list android:opacity="transparent" android:paddingLeft="20dp">
+            <!--<item>
```

Comment:
Why is this block commented? Did you do other tests?

Reply:
yes. just the first version of changes